advanced/Scripts/update.sh

```diff
@@ -47,23 +35,29 @@ is_repo() {
 	# Use git to check if directory is currently under VCS
 	echo -n ":::    Checking $1 is a repo..."
 	cd "${1}" &> /dev/null || return 1
-	git status &> /dev/null && echo " OK!"; return 0 || echo " not found!"; return 1
+	if [[ $(git status --short) ]]; then
```

Comment:
Yeah, as it is now that script isn't `set -e` so we could check retVal and then proceed from there. That was basically an attempt to move from oneliner to full block conditional just to mak sure that the code is solid and there's no room for misinterpreting what the oneliner does. Make it easy for contributors to know what does what on first look.